### ISSUE TITLE
Couldn't find package "@applitools/rendering-grid-client@^1.0.3"

### ISSUE THREAD

**Issue by Strajk** [applitools/eyes.cypress#2]
Hey 👋 

https://github.com/applitools/eyes.cypress/releases/tag/v1.5.12 broke installation for me:

```
 Couldn't find package "@applitools/rendering-grid-client@^1.0.3" required by "@applitools/eyes.cypress@1.5.12" on the "npm" registry.
```

v1.5.11 works well

**Comment by amitzur** [comment]
Indeed, the `@applitools/rendering-grid-client` dependency was published with restricted access.
This is fixed now, so please update to `1.5.13` to get it working.
Thanks for letting us know!

**Comment by Strajk** [comment]
@amitzur works! thx
